docs/configuration.asciidoc

```diff
@@ -1046,6 +1046,32 @@ you must instead set the `LogLevel` for the internal APM logger under the `Loggi
 | `Error`                 | String
 |============
 
+[float]
+[[config-trace-context-ignore-sampled-false]]
+==== `TraceContextIgnoreSampledFalse`
+
+By setting this to `true` the agent will ignore the sampled part of the W3C TraceContext traceparent header when it's false and when the upstream transaction is not coming from a non-Elastic APM agent.
```

Comment:
It might be useful to explain the "normal" flow when Trace Context is received, and then lead into describing what this setting does. In doing so, I think it would provide more context around why it is needed and if it might be relevant to a specific user.

Reply:
Done